lib/cssedit-groups-view.coffee

```diff
@@ -7,26 +7,43 @@ class CsseditGroupsView extends SelectListView
     initialize: (serializeState) ->
         super
         @addClass('overlay from-top task-list')
-        return atom.workspaceView.command 'cssedit-groups:toggle', =>
-            return @.toggle()
+        atom.commands.add 'atom-workspace', 'cssedit-groups:toggle', => @.toggle()
```

Comment:
Just a heads up, this could be `@toggle()` instead of `@.toggle()`. They compile the same way.